docs/reference/commandline/ps.md

```diff
@@ -24,6 +24,7 @@ weight=1
       -q, --quiet=false     Only display numeric IDs
       -s, --size=false      Display total file sizes
       --since=""            Show created since Id or Name, include non-running
+      -F, --format=[]       Pretty-print containers using a Go template
```

Comment:
Hm. Catching up with this PR; I think it's utterly confusing to use an uppercase `-F` here (because other locations use lowercase). 

Could we consider dropping the `-F`? I'm not sure this feature warrants a short flag and it's easier to add a flag later, then it is to deprecate and remove it once added.